migrator/src/main/java/uk/nhs/digital/ps/migrator/model/hippo/Archive.java

```diff
@@ -0,0 +1,24 @@
+package uk.nhs.digital.ps.migrator.model.hippo;
+
+public class Archive extends HippoImportableItem {
+
+    private final String title;
+    private final String summary;
+
+    public Archive(final Folder parentFolder,
+                  final String cmsNodeDisplayName,
+                  final String title,
+                  final String summary) {
```

Comment:
If i'm being extremely picky, your parameter alignment is slightly out of whack 🐵